istresearch/rapidpro
# P4-2269 - Add imo scheme and mode

# For the Reviewer

When this is complete, you should approve the PR via github.

## Summary
Added IMO as a valid pm chat mode, and pm_imo as a valid scheme for the mode. 

#### Release Note
Added IMO as a postmaster chat mode.

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

1. `docker-compose up -d`
2. Ensure you can add a contact with at least one `Postmaster IMO Identifier`
3. The displayed icon for the URN should be the IMO logo.
4. Ensure you can create a postmaster channel with the IMO mode.
